source/future.tex

```diff
@@ -51,6 +51,15 @@
 \rSec1[depr.c.headers]{C standard library headers}
 
 \pnum
+For compatibility with the C standard library\indextext{library!C standard},
+the \Cpp standard library provides headers
+\tcode{<ccomplex>}~(\ref{depr.ccomplex.syn}),
```

Comment:
@jensmaurer: OK, snark alert: If you squint at it right, you could think of "C standard library headers" to be those headers in the standard library that start with "C". Then it would kind of make sense.

Ultimately, *everything* we describe in this document is about C++. The name "C header" is just a mnemonic.